yogstation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -17,6 +17,11 @@
 		meowcount--
 	return ..()
 
+/datum/reagent/consumable/catsip/on_mob_life(mob/living/carbon/C)
+	if(iscatperson(C))
+		C.adjustOrganLoss(ORGAN_SLOT_BRAIN, -1*REM)
+	return ..()
```

Comment:
just merge this all into the proc directly above rather than having it be seperate
also you forgot the /ethanol/ in the pathing anyways

Reply:
Ah, right will do. It was because I was trying to sort out the felinid bit not procing. Thanks